mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_folder_mute_config_test.go

```diff
@@ -101,11 +111,19 @@ resource "google_folder" "test2" {
   parent       = "organizations/%{org_id}"
 }
 
+resource "google_project_iam_binding" "folder_test2_binding" {
+	project = "YOUR_PROJECT_ID"
```

Comment:
Is this example not used in a test? Why are there placeholder values here?

Reply:
addressed